src/paas_app_charmer/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Deprecated entrypoints for charms.
+
+This module can be removed when paas_charm>=2.0
+"""
+
+import warnings
+
+warnings.warn("paas_app_charmer package is deprecated", DeprecationWarning, stacklevel=2)
```

Comment:
Perhaps include a note here on the maintenance window which is that it will be maintained for the lifetime of 22.04 and 24.04

Reply:
And removed in 26.04

Reply:
I added in the module docstring the comment about 22.04 and 24.04.

In the case of the warning for user, I am not sure about the message. I mean, when we release v2 of the library, all v1 may be changed/deprecated so not sure that the extra info will be useful.